Week-01/guessing-game.js

```diff
@@ -42,19 +42,19 @@ dinosaur = 8;
 dinosaur = 999999;
 
 // Q: In the code above, what is the value of `dinosaur`?
-// A:
+// A: -200
```

Comment:
Close again, but `-200` is the initial value of `dinosaur`. It is assigned to `8` on line 41 and then to `999999` on line 42, so the actual final answer is `999999`.